pkgs/applications/science/medicine/aliza/default.nix

```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation {
   src = fetchurl {
     # See https://www.aliza-dicom-viewer.com/download
     urls = [
-      "https://drive.google.com/uc?export=download&id=1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y"
+      "https://www.dropbox.com/s/py0w8kqg3xjvmac/aliza-1.98.57-2.x86_64.rpm?raw=1"
```

Comment:
```suggestion
      "https://drive.google.com/u/0/uc?export=download&confirm=8nbl&id=1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y"
      "https://www.dropbox.com/s/py0w8kqg3xjvmac/aliza-1.98.57-2.x86_64.rpm?raw=1"
```

Does that work?

Reply:
It looks like does not. Here what gets retrieved:

```html
<!DOCTYPE html><html><head><title>Google Drive - Virus scan warning</title><meta http-equiv="content-type" content="text/html; charset=utf-8"/><link href=&#47;static&#47;doclist&#47;client&#47;css&#47;2149812255&#45;untrustedcontent.css rel="stylesheet" nonce="A1EmO2vjV7Mmm7XcnWibgQ"><link rel="icon" href="//ssl.gstatic.com/images/branding/product/1x/drive_2020q4_32dp.png"/><style nonce="A1EmO2vjV7Mmm7XcnWibgQ">#gbar,#guser{font-size:13px;padding-top:0px !important;}#gbar{height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}@media all{.gb1{height:22px;margin-right:.5em;vertical-align:top}#gbar{float:left}}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb4{color:#00c !important}.gbi .gb4{color:#dd8e27 !important}.gbf .gb4{color:#900 !important}
</style><script nonce="8L0K3JUrtsW3rxSjACki1g"></script></head><body><div id=gbar><nobr><a target=_blank class=gb1 href="https://www.google.fr/webhp?tab=ow">Search</a> <a target=_blank class=gb1 href="http://www.google.fr/imghp?hl=en&tab=oi">Images</a> <a target=_blank class=gb1 href="https://maps.google.fr/maps?hl=en&tab=ol">Maps</a> <a target=_blank class=gb1 href="https://play.google.com/?hl=en&tab=o8">Play</a> <a target=_blank class=gb1 href="https://www.youtube.com/?gl=FR&tab=o1">YouTube</a> <a target=_blank class=gb1 href="https://news.google.com/?tab=on">News</a> <a target=_blank class=gb1 href="https://mail.google.com/mail/?tab=om">Gmail</a> <b class=gb1>Drive</b> <a target=_blank class=gb1 style="text-decoration:none" href="https://www.google.fr/intl/en/about/products?tab=oh"><u>More</u> &raquo;</a></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target="_self" href="/settings?hl=en_US" class=gb4>Settings</a> | <a target=_blank  href="//support.google.com/drive/?p=web_home&hl=en_US" class=gb4>Help</a> | <a target=_top id=gb_70 href="https://accounts.google.com/ServiceLogin?hl=en&passive=true&continue=https://drive.google.com/u/0/uc%3Fexport%3Ddownload%26confirm%3D8nbl%26id%3D1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y&service=writely&ec=GAZAMQ" class=gb4>Sign in</a></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div class="uc-main"><div id="uc-dl-icon" class="image-container"><div class="drive-sprite-aux-download-file"></div></div><div id="uc-text"><p class="uc-warning-caption">Google Drive can't scan this file for viruses.</p><p class="uc-warning-subcaption">This file is executable and may harm your computer. <span class="uc-name-size"><a href="https://drive.google.com/open?id=1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y">aliza-1.98.57-2.x86_64.rpm</a> (55M)</span></p><a id="uc-download-link" class="goog-inline-block jfk-button jfk-button-action" href="/uc?export=download&amp;confirm=PUay&amp;id=1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y">Download anyway</a></div></div><div class="uc-footer"><hr class="uc-footer-divider">&copy; 2021 Google - <a class="goog-link" href="//support.google.com/drive/?p=web_home">Help</a> - <a class="goog-link" href="//support.google.com/drive/bin/answer.py?hl=en_US&amp;answer=2450387">Privacy & Terms</a></div></body></html>
```